code/__DEFINES/dcs/signals.dm

```diff
@@ -195,6 +195,10 @@
 #define COMSIG_ATOM_RELAYMOVE "atom_relaymove"
 	///prevents the "you cannot move while buckled! message"
 	#define COMSIG_BLOCK_RELAYMOVE (1<<0)
+///called at the end of /obj/projectile/can_hit_target(): (obj/projectile, direct_target, ignore_loc, cross_failed)
+#define COMSIG_ATOM_CAN_BE_HIT_BY_PROJECTILE "atom_can_be_hit_by_projectile"
```

Comment:
What is this? Is it a check? or a signal that tells you the atom is being hit by a projectile

Reply:
A signal that tells you the atom is about to get hit by a projectile. A last chance for the atom to avoid getting shot.